src/components/ContactInfoSummary.tsx

```diff
@@ -0,0 +1,228 @@
+import { Col, Row, Space, Typography } from 'antd';
+import dateformat from 'dateformat';
+import React from 'react';
+import styled from 'styled-components';
+import { ContactInfo } from '../containers/setContacts/ducks/types';
+import { DEFAULT_IMAGE } from '../utils/copy';
+const { Text } = Typography;
+
+interface ContactInfoSummaryProps {
+  userId: number;
+  info: ContactInfo;
+}
+
+const ContactRow = styled.div`
+  height: 200px;
+  margin-bottom: 16px;
+`;
+
+const ProfilePicture = styled.img`
+  width: 200px;
+  height: 200px;
```

Comment:
Using `vw` and `vh` might help with the responsiveness

Reply:
`vw` and `vh` are awesome, in this case we want the profile picture to have a constant pixel size though. In general I would avoid those units for things like pictures because it makes the layout unpredictable and it can get way too big on large screens. I think a media query or `clamp` would be more appropriate if I wanted a responsive sizing here.